dashboard/serializers.py

```diff
@@ -47,5 +50,6 @@ def serialize(cls, program_enrollment):
             'id': program.id,
             'enrollments': program_cached_enrollments,
             'certificates': program_cached_certificates,
-            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates)
+            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates),
+            'non_learner': has_role(user, NON_LEARNERS)
```

Comment:
this is not good:
our roles are per Programs while this does not verify that the user might not be `Staff` in this program while she is in another one.
I think that the right way to verify this is to create a utility function that checks something like
```
has_role(user, NON_LEARNERS) and Role.objects.filter(user=user, role__in= NON_LEARNERS, program=program).exists()
```